examples/games/padracing/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous
```

Comment:
I thought the `.gitignore` in the root of the repo ought to be sufficient for all subprojects, so that we wouldn't have to add them everywhere.

Reply:
👀 oh yea that shouldn't have been committed!

Should we maybe gitignore any gitignore file in the examples dir? This was generated when I added a platform to the example

Reply:
Haha, sounds good

Reply:
Fixed it!